dev-requirements.txt

```diff
@@ -0,0 +1,2 @@
+flake8
+pytest==4.3.0
```

Comment:
What are dev-requirements? Specific to development? Should the dev install both requirements and dev-requirements?

Reply:
Yes.